test/case/ietf_system/ntp_client/Readme.adoc

```diff
@@ -0,0 +1,24 @@
+=== Simple NTP client test
+==== Description
+Verify NTP client to multiple NTP servers and verify that one get selected
```

Comment:
Grammar, maybe "Verify NTP client with multiple servers, ensure one get selected."

Also, what's the basis of the selection, strata?  Then perhaps say "the best one get selected."

Reply:
All server has same stratum etc, not possible to change in busybox ntpd, so now it just test that "someone" gets selected.

Reply:
OK, that's fine :smiley: